bigbluebutton-html5/imports/ui/components/whiteboard/slide-controls/component.jsx

```diff
@@ -0,0 +1,296 @@
+import React, { Component, PropTypes } from 'react';
+import styles from './styles.scss';
+import Button from '/imports/ui/components/button/component';
+import { callServer } from '/imports/ui/services/api/index.js';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+
+export default class SlideControls extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { sliderValue: 100 };
+
+    this.nextSlide = this.nextSlide.bind(this);
```

Comment:
@OZhurbenko Did you read the documentation? Its a good practice to always bind your event handlers on the constructor.

> We recommend that you bind your event handlers in the constructor so they are only bound once for every instance
> This is better for performance of your application, especially if you implement shouldComponentUpdate() with a shallow comparison in the child components.

We should always try to deliver the best code possible, not just one that works.